ci.env

```diff
@@ -0,0 +1,25 @@
+CI=true
+DOCKER_GATEWAY_HOST=172.17.0.1
+ETHEREUM_RPC_URL=https://mainnet.infura.io/v3/275c8fc2ec294d6aac6bfc881d9a9dff
```

Comment:
Who's Infura key is this? Should probably not be exposed to public in the repo since its rate limited. Putting it in GH secrets likely best option

Reply:
This is ours. I check on usage once in a while and we've never come close to hitting the limits. But yeah, we can move it to a secret.

Reply:
@levity I'll leave this to you